src/lib/mina.ts

```diff
@@ -290,6 +318,9 @@ function LocalBlockchain({
     applyJsonTransaction(json: string) {
       return ledger.applyJsonTransaction(json, String(accountCreationFee));
     },
+    fetchEvents(publicKey: PublicKey, tokenId: Field): any {
+      return events[publicKey.toBase58()][Ledger.fieldToBase58(tokenId)];
```

Comment:
could we return an empty array here instead of failing / returning `undefined`, if there are no events stored for that account / tokenId yet?
Also, I know we are bad at typing events right now, but I'd prefer `any[]` as the return type over `any`

Reply:
Another thing: `tokenId` should be optional and use the default id by default, like `getAccount`